test/test_bson_corpus.py

```diff
@@ -220,7 +232,7 @@ def create_tests():
         if test_suffix == 'multi-type':
             # Special case in TestBSONCorpus.
             continue
-        with open(filename) as bson_test_file:
+        with codecs.open(filename, encoding='utf-8') as bson_test_file:
```

Comment:
I cannot reproduce the problem on arch, even on a spawn host using the evergreen shell script. The problem seems to be coming from calling `fp.read()` inside of `json.load`. This line allows the tests to pass: https://evergreen.mongodb.com/version/58ae3b7f3ff1225d7302754b